scripts.js

```diff
@@ -12,6 +12,9 @@ var hexes = document.querySelectorAll(".box-details > p");
 // Selecting all the icon span with material-symbols-outlined class, instead of selecting each one separetly.
 var locks = document.querySelectorAll(".material-symbols-outlined");
 
+// delete query Selector
+savedPaletteSection = document.querySelector(".saved-palettes")
+
```

Comment:
Looks like we need to add `var` at the beginning of line 16 since this is where we are creating the variable